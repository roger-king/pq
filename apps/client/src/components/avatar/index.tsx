import { AvatarIcon } from './avatar';

export default AvatarIcon;
