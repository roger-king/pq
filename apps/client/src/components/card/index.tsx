export * from './question';
export * from './game';
