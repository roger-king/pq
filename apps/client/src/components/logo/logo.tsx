import React from 'react';
import { Box, Heading } from 'grommet';

export const Logo: React.FC = () => {
  return (
    <Box background="brand" pad="small">
      <Heading margin="none" level="2">
        PQ
      </Heading>
    </Box>
  );
};
