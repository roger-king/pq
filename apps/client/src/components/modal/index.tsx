import React, { PropsWithChildren } from 'react';
import { Box, Heading, Layer } from 'grommet';

export interface BaseModalProps {
  title: string;
  onClose(): void;
}

const Modal: React.FC<PropsWithChildren<BaseModalProps>> = (props: PropsWithChildren<BaseModalProps>) => {
  const { children, title } = props;

  return (
    <Layer modal>
      <Box fill align="center" justify="center" pad="medium" background="brand">
        <Box direction="row" align="center" justify="between" fill>
          <Heading level="3">{title}</Heading>
        </Box>
        {children}
      </Box>
    </Layer>
  );
};

export default Modal;
