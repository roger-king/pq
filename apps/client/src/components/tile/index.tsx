import { Tile } from './tile.component';

export default Tile;
