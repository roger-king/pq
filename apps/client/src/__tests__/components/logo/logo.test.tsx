import React from 'react';
import renderer from 'react-test-renderer';

import { Logo } from '../../../components/logo';

describe('<Logo />', () => {
  test('should display default logo', async () => {
    const tree = renderer.create(<Logo />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
