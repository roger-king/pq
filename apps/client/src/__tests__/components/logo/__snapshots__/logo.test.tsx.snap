// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should display default logo 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 cIkaNd"
>
  <h2
    className="StyledHeading-sc-1rdh4aw-0 frzykv"
  >
    PQ
  </h2>
</div>
`;
